src/main/java/org/apache/commons/lang3/ArrayUtils.java

```diff
@@ -4609,17 +4609,8 @@ public static int lastIndexOf(final short[] array, final short valueToFind, int
      * (index &lt; 0 || index &gt;= array.length), or if the array is {@code null}.
      * @since 2.1
      */
-    public static boolean[] toPrimitive(final Boolean[] array) {
```

Comment:
You cannot delete public APIs, that breaks binary compatibility within a major release.

Reply:
Oh. Fuck me!

Was an accident. I will fix.

Reply:
I think you were looking at some outdated version of the request.

This method starts on the line 8886 in the current pull request and also on the current master.

The other possibility that I am totally missing something. One thing is for sure: I would never intentionally suggest the deletion or change of an API method.

Could you have a look at it again?